pkg/api/operators/apis/experiment/v1alpha2/constants.go

```diff
@@ -27,4 +27,10 @@ const (
 
 	// Default value of Spec.TemplatePath
 	DefaultTrialTemplatePath = "defaultTrialTemplate.yaml"
+
+	// Name of the configMap for Trial templates
+	TrialTemplatesName = "trial-template"
+
+	// Name of the configMap for MetricsCollector templates
+	MetricsCollectorTemplatesName = "metrics-collector-template"
```

Comment:
Is this the default metric config map similar to trial?

Reply:
Yes, I will rename it, like for Trial template